suite/tests/client-interface/mangle_suspend.dll.c

```diff
@@ -129,6 +126,51 @@ suspend_func()
 #endif
 }
 
+static void
+suspend_test_2_func()
+{
+#ifdef X86_64
+    void **drcontexts = NULL;
+    uint num_suspended;
+    if (dr_suspend_all_other_threads(&drcontexts, &num_suspended, NULL)) {
+        if (num_suspended != 1)
+            CHECK(false, "num_suspended unexpected!");
+        dr_mcontext_t mc = { sizeof(mc), DR_MC_INTEGER | DR_MC_CONTROL };
+        if (!dr_get_mcontext(drcontexts[0], &mc))
+            CHECK(false, "dr_get_mcontext failed!");
+        /* Check xip such that we only perform the check if we are in the loop
+         * body of the test, in order to avoid races. The magic values 16 and 42
+         * are byte counts to compute the bounds of the asm loop in the test.
```

Comment:
Please include a comment in the test about updating here when changing its code.

Reply:
Done.